changelog.d/20231025_205335_30907815+rjmello_gcengine_status.rst

```diff
@@ -0,0 +1,4 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added support for endpoint status reports when using ``GlobusComputeEngine``.
```

Comment:
A few of the more useful keys would be more informative to list in the CL I think (though not all of them because that'd be too verbose/not definitive).  ie. "such as 'total_workers', 'idle_workers', 'pending_tasks' etc"

Reply:
Good call.